tokio/src/runtime/context.rs

```diff
@@ -47,6 +47,19 @@ cfg_rt_core! {
     }
 }
 
+cfg_syscall! {
+    use std::sync::Arc;
+    use crate::syscall::Syscalls;
+
+    #[allow(dead_code)] // Unused for now, gated by the unstable flag
+    pub(crate) fn syscalls() -> Option<Arc<dyn Syscalls>> {
+        CONTEXT.with(|ctx| match *ctx.borrow() {
+            Some(ref ctx) => ctx.syscalls.clone(),
+            None => None
+        })
```

Comment:
Nit/take it or leave it: I think this could be simplified by using `map`:
```suggestion
        CONTEXT.with(|ctx| match ctx.borrow().map(|ctx| ctx.syscalls.clone()))
```